src/app/client/src/app/app.component.html

```diff
@@ -127,7 +127,7 @@
 </div>
 <app-footer [layoutConfiguration]="layoutConfiguration" *ngIf="!hideHeaderNFooter"></app-footer>
 
-<app-modal-wrapper *ngIf="sessionExpiryInterceptor.sessionExpired"
+<!-- <app-modal-wrapper *ngIf="sessionExpiryInterceptor.sessionExpired"
```

Comment:
@mithun30052001 why comment on this line?

Reply:
The comment is not needed and is removed @princegupta1131